code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -78,6 +78,12 @@
 	description = "Rare and experimental particles, that apparently swap the user's body with one from an alternate dimension where it's completely healthy."
 	taste_description = "science"
 
+/datum/reagent/medicine/adminordrazine/strangevodka
+	name = "Strange Vodka"
+	description = "An ancient mixture of extremely strong and rare reagents. Has the power to rapidly repair any and all wounds, at the cost of getting the user very, very drunk."
+	taste_description = "the motherland"
+	var/boozepwr = 200
```

Comment:
😳

Reply:
an i oop

sorry i didnt know if i was supposed to clear this with you, since this isn't a chemical you can find large amounts of / produce any of

Reply:
this doesn't even work lol

Reply:
explain

Reply:
boozepwr does absolutely nothing to this reagent because you just defined a var and have no code there

if you want it to do something it has to derive from this type

https://github.com/tgstation/tgstation/blob/783a6dd5eb2b0470e1055c34e86db3414a8cbf54/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm#L37

not that i'd like your boozepwr because according to this chart you would just literally die like instantly

also not that i see a reason to even have this reagent, it's literally just the admin chem and does nothing different and the container does all the flavortext and contains the ethanol that gets you drunk